instrumentation-api-annotation-support/src/main/java/io/opentelemetry/instrumentation/api/annotation/support/MethodSpanAttributesExtractor.java

```diff
@@ -54,18 +59,81 @@ protected void onStart(AttributesBuilder attributes, REQUEST request) {
   protected void onEnd(
       AttributesBuilder attributes, REQUEST request, @Nullable RESPONSE response) {}
 
-  private static class MethodSpanAttributeBinder extends BaseAttributeBinder {
-    private final ParameterAttributeNamesExtractor parameterAttributeNamesExtractor;
+  /**
+   * Creates a binding of the parameters of the traced method to span attributes.
+   *
+   * @param method the traced method
+   * @return the bindings of the parameters
+   */
+  private AttributeBindings bind(Method method) {
+    AttributeBindings bindings = EmptyAttributeBindings.INSTANCE;
+
+    Parameter[] parameters = method.getParameters();
+    if (parameters == null || parameters.length == 0) {
```

Comment:
```suggestion
    if (parameters.length == 0) {
```

Don't think can be null